tests/wpt/meta/encoding/streams/decode-utf8.any.js.ini

```diff
@@ -6,35 +6,6 @@
 
 [decode-utf8.any.worker.html]
   expected: ERROR
```

Comment:
Note to self: error is from https://github.com/servo/servo/blob/0e4b2bfef0775f5b266f563d7ca699fd6837fb27/tests/wpt/tests/common/sab.js#L1

"sabConstructor is not a constructor".